### PR TITLE
Feature 2881 use brokerage fee

### PR DESCRIPTION
**Pull request by Adalyat**
#### Description

Instead of using BitfinexFeeModel we can use fee information received from brokerage on trade update.

#### Related Issue

#2881 

#### Motivation and Context

It allows to keep fee accurately. Even brokerage change their fee rates we still can have relevant information without FeeModel call update

#### How Has This Been Tested?

Order execution, compare "theirs" and "ours" values

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Non-functional change (xml comments/documentation/etc)

#### Checklist:

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [ ] I have added tests to cover my changes. 
- [x] All new and existing tests passed.
- [x] My branch follows the naming convention `bug-<issue#>-<description` or `feature-<issue#>-<description>`

### CODE REVIEW

File: Brokerages/Bitfinex/BitfinexBrokerage.Messaging.cs [hunk 1]
```diff
@@ -561,13 +561,10 @@ private void EmitFillOrder(string[] entries)
...
                 var updTime = Time.UnixTimeStampToDateTime(double.Parse(entries[3], NumberStyles.Float, CultureInfo.InvariantCulture));
-                var orderFee = OrderFee.Zero;
-                if (fillQuantity != 0)
-                {
-                    var security = _securityProvider.GetSecurity(order.Symbol);
-                    orderFee = security.FeeModel.GetOrderFee(
-                        new OrderFeeParameters(security, order));
-                }
+                var orderFee = new OrderFee(new CashAmount(
+                        Math.Abs(decimal.Parse(entries[9], NumberStyles.Float, CultureInfo.InvariantCulture)),
```
Comment (jaredbroad) [inline Brokerages/Bitfinex/BitfinexBrokerage.Messaging.cs]: Thanks @Adalyat! Getting close to having Bitfinex live. 

Can we get a fee for a currency we don't have an asset for? Let's check what this is before passing back. What happens when the cashbook get an unknown currency?

Comment (Adalyat) [reply Brokerages/Bitfinex/BitfinexBrokerage.Messaging.cs]: Hello!
Currenty this situation is handled by EnsureCurrencyDataFeeds method and throw exception on init.
tested for this:
Account currency: ETH
Trading: BTCUSD


 
Is it expected behavior?

Comment (StefanoRaggi) [reply Brokerages/Bitfinex/BitfinexBrokerage.Messaging.cs]: @Adalyat It's a Lean bug (unrelated to Bitfinex brokerage), opened a new issue here: https://github.com/QuantConnect/Lean/issues/2885

Comment (Adalyat) [reply Brokerages/Bitfinex/BitfinexBrokerage.Messaging.cs]: @StefanoRaggi  and @jaredbroad 
Just tested another case:

```
SetAccountCurrency("USD");
AddCrypto("ETHBTC", Resolution.Minute, Market.Bitfinex);
```
everything works fine and ETH fee currency handled properly

**Pull request merged**